Makefile

```diff
@@ -242,7 +242,7 @@ CPU_COUNT = $(shell cat /proc/cpuinfo | grep processor | wc -l )
 	# -> filestash config at $(TMP_PATH_TO_FILESTASH_CONFIG)
 	@$(shell \
 		export TMP_PATH_TO_FILESTASH_CONFIG="${TMP_PATH_TO_FILESTASH_CONFIG}" && \
-		docker-compose --env-file .env --file services/docker-compose-ops.yml --log-level=DEBUG config > $@ \
+		docker-compose --env-file .env --file services/docker-compose-ops.yml --log-level=DEBUG config | sed -E "s/cpus: ([0-9\\.]+)/cpus: '\\1'/" > $@ \
```

Comment:
See: https://github.com/docker/compose/issues/7771#issuecomment-765243575

Reply:
As pointed out by the post above, it's an issue with `docker-compose` that is not getting fixed. Since It is currently frozen at version `1.29.2`. We have not upgraded to version 2.x.x which I'm not sure that if fixes this issue.

Reply:
I see, so then I propose that you add a comment in this change so that we can follow up when we do the upgrade.
IMO patching the docker-compose this way should only be done in exceptional circumstances (this could be one of them!).

Regarding the regex... It realize now that I do not understand how really ``sed``  works because 
```cmd
echo cpus:    "4.3" | sed -E "s/cpus: ([0-9\\.]+)/cpus: '\\1'/"
cpus: '4.3'

echo "cpus:    '4.3'" | sed -E "s/cpus: ([0-9\\.]+)/cpus: '\\1'/"
cpus:    '4.3'
```

despite the fact that the regex clearly misses spaces ``\w`` and the quotes as shown in https://regex101.com/r/a99Y5z/1